src/events/mouse.js

```diff
@@ -929,4 +969,44 @@ p5.prototype._onwheel = function(e) {
   }
 };
 
+/**
+ * <p>The function <a href="#/p5/requestPointerLock">requestPointerLock()</a>
+ * locks the pointer to its current position and makes it invisible.
+ * Use deltaX and deltaX to get the difference the mouse was moved since
+ * the last call of draw</p>
+ * <p>Note that not all browsers support this feature</p>
+ *
+ * @method requestPointerLock
+ * @returns {void}
+ * @example
+ * <div class="notest">
+ * <code>
+ * //click the canvas to lock the pointer
+ * //press ESC to exit lock mode
+ * function draw() {
+ *   background(237, 34, 93);
+ *   fill(0);
```

Comment:
the `fill()` line here is unnecessary since nothing is drawn. it can be removed.

Reply:
https://github.com/processing/p5.js/pull/3804/commits/2fcfa818300f93b63b3363efc8670254fe2e0fc6 fill removed